dog_lover/components/topBar.tsx

```diff
@@ -0,0 +1,262 @@
+import React, { FunctionComponent } from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import Grid from "@mui/material/Grid";
+import { NextRouter, useRouter } from "next/router";
+import { useVariableOpenContext } from "../src/breedContext";
+
+const TopBar: FunctionComponent = () => {
+  const router: NextRouter = useRouter();
+  const isOpened = useVariableOpenContext();
+
+  const oneClickRandomDog = () => {
+    const randomDogButton = document.getElementById("dogGenerator");
+    if (randomDogButton) {
+      randomDogButton.click();
+    }
+  };
+  const oneClickBreedsList = () => {
+    const breedSelectorButton = document.getElementById("whoLetTheDogsOut");
```

Comment:
Nice naming here! LGTM!

Reply:
Thanks! Also thank you for the "LGTM", I had no idea what it meant but now I do 😂